packages/gatsby-source-nacelle/gatsby-node.js

```diff
@@ -0,0 +1,245 @@
+const { ensureDir } = require('fs-extra');
+const { print } = require('gatsby/graphql');
+const {
+  sourceAllNodes,
+  createSchemaCustomization,
+  compileNodeQueries,
+  readOrGenerateDefaultFragments,
+  buildNodeDefinitions,
+  loadSchema,
+  createDefaultQueryExecutor
+} = require('gatsby-graphql-source-toolkit');
+require('dotenv').config();
+
+const CHUNK_SIZE = 100;
+const fragmentsDir = process.cwd() + `/gql-fragments`;
+
+const PaginateNacelle = {
+  name: 'NacellePagination',
+  expectedVariableNames: ['first', 'after'],
+  start() {
+    return {
+      variables: { first: CHUNK_SIZE, after: undefined },
+      hasNextPage: true
+    };
+  },
+  next(state, page) {
+    return {
+      variables: { first: CHUNK_SIZE, after: page.nextToken },
+      hasNextPage: Boolean(page.nextToken && page.items.length === CHUNK_SIZE)
+    };
+  },
+  concat(acc, page) {
+    return {
+      ...acc,
+      items: {
+        ...acc.items,
+        ...page.items
+      },
+      nextToken: page.nextToken
+    };
+  },
+  getItems(pageOrResult) {
+    return pageOrResult.items;
+  }
+};
+
+async function createSourcingConfig(gatsbyApi, pluginOptions) {
+  const { verbose } = pluginOptions;
+
+  // Extract credentials from plugin options
+  const nacelleSpaceId =
+    pluginOptions.nacelle_space_id ||
```

Comment:
Just out of curiosity -- how come this is snake_case to begin with?

Reply:
I think it was a subconscious effort to promote readability, since GraphQL is a little awkward in camelCase form, e.g.:

<img width="429" alt="Screen Shot 2020-09-10 at 10 42 04 PM" src="https://user-images.githubusercontent.com/5732000/92847150-dae10980-f3b6-11ea-8aac-c92944c8be4f.png">

I added a function that will ingest any case thrown at it (camel, kebab, snake, pascal) just for kicks 😄

![get-plugin-option](https://user-images.githubusercontent.com/5732000/92847642-622e7d00-f3b7-11ea-8e4c-ec540feea5cc.png)

Reply:
Hm. Thoughts on making it all just `camelCase` in our examples? As mentioned earlier, I think the flexibility is cool, but also we should just be pushing people in the direction we want them to go in, which should be `camelCase` since that's the de factor for JS casing.

Specifically looking at these files: 
```
examples/gatsby/gatsby-config.js
packages/gatsby-source-nacelle/README.md
packages/gatsby-source-nacelle/gatsby-node.js
```

Reply:
No problem, I changed the examples in those files from `kebab-case` to `camelCase` in https://github.com/getnacelle/nacelle-react/pull/8/commits/d2e48dda96618ae39ebb2b9b3fe8b7f3a7c2a17e